api/api/viewsets.py

```diff
@@ -29,11 +34,20 @@ def _get_status_from_query(params):
         serializer = serializers.IdentitySerializer(data=params)
         serializer.is_valid(raise_exception=True)
 
+        email = serializer.validated_data.pop('email')
         identity = Identity(**serializer.validated_data)
         status = Status()
 
         fetch_and_update_registration(identity, status)
 
+        if email:
+            voter, created = Voter.objects.update_or_create(
+                email=email,
+                defaults=serializer.validated_data,
+            )
+            if not created:
+                log.warning(f"Updated exiting voter: {voter}")
```

Comment:
We'll probably need to deal with this at some point otherwise you'll be able to update someone else's account.